r2443 | andreas.stieger@gmx.de | 2014-10-21 17:42:56 -0400 (Tue, 21 Oct 2014) | 7 lines

Follow-up to r2419: Correct comment style

* test/test_buckets.c
  (deflate_compress): Correct comment style not supported by strict
  compilers, fails on /branches/1.3.x

Index: test/test_buckets.c
===================================================================
--- test/test_buckets.c	(revision 2442)
+++ test/test_buckets.c	(revision 2443)
@@ -1323,9 +1323,9 @@
 
     /* The largest buffer we should need is 0.1% larger than the
        uncompressed data, + 12 bytes. This info comes from zlib.h.
+       buf_size = orig_len + (orig_len / 1000) + 12;
        Note: This isn't sufficient when using Z_NO_FLUSH and extremely compressed
        data. Use a buffer bigger than what we need. */
-//    buf_size = orig_len + (orig_len / 1000) + 12;
     buf_size = 100000;
 
     write_buf = apr_palloc(pool, buf_size);
